src/Cynthia.Card/src/Cynthia.Card.Server/Services/GwentGameService/GwentServerService.cs

```diff
@@ -368,40 +368,23 @@ public void InvokeGameOver(GameResult result, bool isOnlyShow, bool isCountMMR)
 
         public int CalculateMMR(int myMMR, int enemyMMR, bool isWin, bool isDraw)
         {
-            int k = CalculateK(myMMR);
-            double s = isDraw ? 0.5 : (isWin ? 1 : 0);
-            int diff = Math.Max(enemyMMR - myMMR, -50);
-            double e = 1 / (1.0 + Math.Pow(10, diff / 400.0));
-            double eta = CalculateEta(s, e, myMMR);
```

Comment:
Hmm, also why you remove the impact of eta

Reply:
i do calculate eta equivalent. its in this line:
double expected = 1 / (1.0 + Math.Pow(10, diff / scale));
i believe its cleaner. al i did (other then k) is talked in great detail here:
https://www.omnicalculator.com/sports/elo#elo-calculator-in-practice-elo-rating-in-a-chess-tournament

Again i believe mmr shouldnt impact k - its practical in chess when higher rated players play each other all the time. Not here.

I didnt delete calculate eta function as i wasnt sure if its not used elsewhere - i can ofcourse